### DISCUSSION TITLE
Should we keep the `Do not copy code from other sources` in the PR template?

### DISCUSSION THREAD

**Discussion by Panquesito7** [julelang/jule#45] [Discussions]
Jule's an open-source project, there are other tons of open-source projects out there. I don't consider it wrong if created PRs take code from another **open-source project**, after all, that's what open-source is all about.
If we allow this though, we'll have to specify that credit should be given according to the project's license.

Let me know your thoughts. Thanks. 🙂

**Comment by mertcandav** [comment]
Yes, you are right.

Actually, we can update this condition instead of removing it directly. For example, it can be arranged such as informing about the license conditions of the relevant code, adding information about the status of following the relevant license conditions or etc.

**Reply by Panquesito7** [reply]
Alrighty! I will work on this. 🙂
